cmd/rook/operator.go

```diff
@@ -69,13 +71,20 @@ func startOperator(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func getClientset() (*kubernetes.Clientset, error) {
+func getClientset() (kubernetes.Interface, apiextensionsclient.Interface, error) {
 	// create the k8s client
 	config, err := rest.InClusterConfig()
 	if err != nil {
-		return nil, fmt.Errorf("failed to get k8s config. %+v", err)
+		return nil, nil, fmt.Errorf("failed to get k8s config. %+v", err)
 	}
 
-	c, err := kubernetes.NewForConfig(config)
-	return c, err
+	clientset, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return nil, nil, fmt.Errorf("failed to create k8s clientset. %+v", err)
+	}
+	apliExtClientset, err := apiextensionsclient.NewForConfig(config)
+	if err != nil {
```

Comment:
No. This is just creating the client. Only if we use it to interact the server is when we would get an error.